Apps/Sandcastle/gallery/development/Shadows.html

```diff
@@ -219,7 +219,7 @@
     size : 1024,
     modelOptions : ['Wood Tower', 'Cesium Air', 'Cesium Man', 'Transparent Box', 'Shadow Tester', 'Shadow Tester 2', 'Shadow Tester 3', 'Shadow Tester 4', 'Shadow Tester Point'],
     model : 'Shadow Tester',
-    locationOptions : ['Exton', 'Everest', 'Pinnacle PA', 'Seneca Rocks', 'Half Dome'],
+    locationOptions : ['Exton', 'Everest', 'Pinnacle PA', 'Seneca Rocks', 'Half Dome', 'Seattle 3D Tiles', 'NYC 3D Tiles'],
```

Comment:
Yes, because it helps to see real use cases inside `development/Shadows.html`. But I can see how this is a problem. I'll switch to using the test 3D Tiles sets we have and change to using Seattle/etc locally.